lib/themes/dosomething/paraneue_dosomething/js/components/Slideshow.js

```diff
@@ -29,9 +46,11 @@ class Slideshow extends React.Component {
 
     return (
       <div className="slideshow">
-        {slides[this.state.activeSlide]}
+        <ReactCSSTransitionGroup className="slideshow__carousel" transitionName="slide" component="div" transitionEnterTimeout={10000} transitionLeaveTimeout={10000}>
```

Comment:
I made the timeout `10000` so it's easier to see how it was behaving by moving it slowly. Change this to around `500` once rest of it figured out :)